doozerlib/cli/detect_embargo.py

```diff
@@ -141,6 +164,31 @@ def detect_embargoes_in_nvrs(runtime: Runtime, nvrs: List[str]):
     embargoed_builds = [b for b in builds if b["id"] in embargoed_build_ids]
     return embargoed_builds
 
+def detect_embargoes_in_images(runtime: Runtime, images: List[str], version: str):
+    """ Check whether latest build of image in given images list become embargoes
+    :param runtime: the runtime
+    :param images: list of images name
+    :return: list of Brew build dicts that have embargoed fixes
+    """
+    runtime.logger.info(f"Fetching {len(images)} builds from Brew...")
+    brew_session = runtime.build_retrying_koji_client()
+    detector = bs_detector.BuildStatusDetector(brew_session, runtime.logger)
+    embargoed_builds = []
+    for image in images:
+        pattern = image + '-container-v' + version
+        results = brew_session.search(pattern, 'build', 'regexp', {'limit': 2, 'order': '-id'})
```

Comment:
Hmm... I think ordering by id might not be 100% correct. Maybe we should order by the time when a build is tagged in.

Reply:
I tried to use time but seems brew don't support such query options